backend/dataall/modules/metadata_forms/db/enums.py

```diff
@@ -20,7 +20,8 @@ class MetadataFormEntityTypes(GraphQLEnumMapper):
     OrganizationTeams = 'Organization Team'
     Environments = 'Environment'
     EnvironmentTeams = 'Environment Team'
-    Datasets = 'Dataset'
+    S3Datasets = 'S3-Dataset'
+    RDDatasets = 'RD-Dataset'
```

Comment:
value must be `Redshift-Dataset` if matching with the path used on FE Client ++ would suggest changing key as well so

```
RedshiftDatasets = 'Redshift-Dataset'
```

Reply:
Sorry